share/picongpu/examples/atomicPhysics/include/picongpu/param/species.param

```diff
@@ -46,14 +46,18 @@ namespace picongpu
     /* define which interpolation method is used to interpolate fields to particle*/
     using UsedField2Particle = FieldToParticleInterpolation<UsedParticleShape, AssignedTrilinearInterpolation>;
 
-/*! select current solver method -----------------------------------------------
- * - currentSolver::Esirkepov<SHAPE>  : particle shapes - CIC, TSC, PCS, P4S (1st to 4th order)
- * - currentSolver::VillaBune<>       : particle shapes - CIC (1st order) only
- * - currentSolver::EmZ<SHAPE>        : particle shapes - CIC, TSC, PCS, P4S (1st to 4th order)
+/*! select current solver method
+ * - currentSolver::Esirkepov< SHAPE, STRATEGY > : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
+ * - currentSolver::EmZ< SHAPE, STRATEGY >       : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
+ * - currentSolver::EZ< SHAPE, STRATEGY >        : same as EmZ (just an alias to match the currently used naming)
  *
- * For development purposes: ---------------------------------------------------
- * - currentSolver::EsirkepovNative<SHAPE> : generic version of currentSolverEsirkepov
- *   without optimization (~4x slower and needs more shared memory)
+ * STRATEGY (optional):
```

Comment:
This `STRATEGY` option was added to this PR. Might be better to introduce this in a different PR.

Reply:
Sorry - This is already introduced in all other examples but was missing the the `atomicPhysics` example 
cc'ing @BrianMarre

Reply:
is fine according the @psychocoderHPC (discussed offline) - will be added with this PR